### DISCUSSION TITLE
Does Ezra Project have UI support for OSIS transliterated forms that use the xlit attribute of the w element?

### DISCUSSION THREAD

**Discussion by DavidHaslam** [ezra-bible-app/ezra-bible-app#246] [Q&A] [answered]
## Transliterated forms
Does Ezra Project have UI support for OSIS transliterated forms that use the **xlit** attribute of the **w** element?
_If so, how are these rendered?_

## Cherokee New Testament
A suitable test module is **Che1860** that was recently updated.
The main text is Cherokee syllabics. The xlit text is the transliteration to the Latin alphabet.
**OSISXlit** is the only **GlobalOptionFilter** key.

## Masoretic Text parallel to the Samaritan Pentateuch
Biblical Hebrew module **SPMT** has **OSIS(Strongs|Morph|Ruby|Xlit|Enum)**.
This could be useful as a text candidate for displaying the various filter combinations.

**Comment by tobias-klein** [comment, marked as answer]
> Does Ezra Project have UI support for OSIS transliterated forms that use the **xlit** attribute of the **w** element?
> _If so, how are these rendered?_

No, the `xlit` attribute is currently not handled in any special way.

What should a frontend do exactly with the xlit attribute? Are there other frontends that do have support for this?
